lib/sles4sap/sap_deployment_automation_framework/deployment.pm

```diff
@@ -562,8 +563,28 @@ sub prepare_sdaf_project {
     assert_script_run("cd $deployment_dir");
     assert_script_run('mkdir -p ' . log_dir());
 
+    # Calculate SDAF version used for deployment and picks latest -1
+    # SDAF_GIT_AUTOMATION_BRANCH variable will override calculated value
+    my $branch = get_var('SDAF_GIT_AUTOMATION_BRANCH', '');
+    if (!$branch || $branch eq 'latest') {
+        my $tags = script_output("curl -s https://github.com/Azure/sap-automation/tags | grep /releases/tag/ | grep data-view-component | cut -d '/' -f6 | cut -d '\"' -f1");
+        record_info("Releases: $tags");
+        my @releases = split('\n', $tags);
+        my $branch_expected = $releases[1];
+        if ($branch eq 'latest') {
+            $branch_expected = $releases[0];
+        }
+        my $branch_er = version->new('v3.11.0.3');
```

Comment:
What does `branch_er` stand for?

Reply:
It is a branch missing features and our test cases will be failed if using this branch.
I did not added comments as I thought the `die` msg is enough.

Reply:
Added description on this.